addons/mail/tests/test_mail_composer.py

```diff
@@ -78,3 +81,65 @@ def test_mail_mass_mode_compose_with_mso(self):
             values[self.partner_employee.id]['body_html'],
             'We must preserve (mso) comments in email html'
         )
+
+    @users('employee')
+    def test_mail_attachment_to_download_link(self):
+        """ Test that when mail size exceed the max email size limit, attachment are turned into download links added
+        at the end of the email content. """
+        attachment_size_in_bytes = 1024 * 128
+        datas = base64.b64encode(bytes(''.join('.' for _ in range(0, attachment_size_in_bytes)), 'utf-8'))
```

Comment:
I really think it should work like this (without the encoding)
```suggestion
        datas = base64.b64encode(''.join('.' for _ in range(0, attachment_size_in_bytes).encode())
```

(same for other occurences)

Reply:
I have replaced bytes('xx', 'utf-8') by 'xx'.encode() as it is equivalent and easier.